docker/deploy.bat

```diff
@@ -0,0 +1,2 @@
+docker-compose -p our_dspace -f deploy.yml up -d
+docker-compose -p our_dspace -f cli.yml run --rm dspace-cli create-administrator -e test@test.edu -f admin -l user -p admin -c en
```

Comment:
I use `pause` at the end of batch so I see what happend once finished

Reply:
added